README.md

```diff
@@ -1,32 +1,212 @@
+* Replace `<auxiliary-graph-rpc>` with auxiliary rpc graph admin endpoint.
 * Replace `<db-path>` with the database path. It is the path for `sqlite` database. If not provided,it would create it.
 * `--force` option is used to forcefully override facilitator config. It is optional parameter.
-	
-## Facilitator Start
 
-Facilitator start command will start the facilitator. 
+
+Facilitator init can be also be done with `--gateway-config <gateway-config>` option. Gateway config i.e. `<gatewayaddress>.json` can be found in [here](https://github.com/mosaicdao/mosaic-chains/tree/develop/chains) inside `<origin-chain>/<auxiliary-chain>/<gateway-address.json`.
+
+Replace <gateway-config> with file location where gateway config is present.
+
+
+This command will generate facilitator config file which is needed to start facilitator. 
+
+
+**5. Set ENV variables**: Facilitator init commands create worker keys which are encrypted with password. Enviornment variables need to be set to unlock the encrypted keystore file. 
+
+Set below env variables after replacing address and password.
+```
+export MOSAIC_ADDRESS_PASSW_<Address>=<origin-password>
+export MOSAIC_ADDRESS_PASSW_<Address>=<auxiliary-password>
+```
+Above variables will also be produced with the output of `facilitator init` command.
+
+**6. Fund facilitator workers for gas and bounty**: Workers created in facilitator init step needs to be funded for gas and bounty on both chains.
+
+   *Gas on origin*: Facilitator needs to pay for gas on origin chain. Origin worker address created in `facilitator init` step must be funded to pay for transaction fee. 
+   
+   There are various ways to fund worker on origin chain. For testnet any public faucet can be used. Below web3 transaction can also be done, if there is fund in an existing key.
+   
+      
+      web3.eth.sendTransaction(
+      {
+         from:<funder_address>, 
+          to: <origin_worker>,
+          value:<Fund_in_wei
+      })
+      
+   *Gas on auxiliary*: Facilitator also needs to pay for transaction fee on auxiliary chain. For testnet [mosaic faucet](https://github.com/mosaicdao/faucet) can be used to get fund. Alternatively, OST on value chain can also be converted to base token on auxiliary chain in order to pay for gas.
+   
+   *Value token for bounty*: Facilitator needs to stake bounty to gateway/co-gateway in order to perform stake & mint and redeem & unstake. Bounty token on the origin chain for existing mosaic chain is simple token and on the auxiliary chain bounty is base token. 
+   
+   On testnet, bounty can be funded to workers using [mosaic faucet](https://github.com/mosaicdao/faucet).
+   
+**7. White list workers**: origin and auxiliary workers should be whitelisted in stakepool and redeem pool contracts respectively. Below commands will whitelist the workers. 
+
+   1. Clone git repository:
+   ```
+       git clone https://github.com/mosaicdao/mosaic-chains.git
```

Comment:
we asked the user to install `mosaic-chains` globally above. Have them clone it again here might be a problem

Reply:
It's for the whitelist-worker script. Any other idea?